setup.cfg

```diff
@@ -41,6 +41,7 @@ include_package_data = true
 zip_safe = false
 install_requires =
     asgiref >= 3.2.10
+    colorama; sys_platform == "win32"
```

Comment:
Can we drop this please? Ultimately we'd like to keep the list of `install_require` down to **required** packages.